infrastructure/docker/Dockerfile-ray-qiskit

```diff
@@ -1,9 +1,19 @@
 ARG IMAGE_PY_VERSION=py39
-FROM rayproject/ray:2.4.0-$IMAGE_PY_VERSION
 
+FROM rayproject/ray:2.4.0-$IMAGE_PY_VERSION AS ray-node-amd64
 USER $RAY_UID
-
 COPY --chown=$RAY_UID:$RAY_UID ./client ./qs
 RUN cd ./qs && pip install .
 RUN cd ../ 
 RUN rm -r ./qs
+
+FROM rayproject/ray:2.4.0-$IMAGE_PY_VERSION-aarch64 AS ray-node-arm64
```

Comment:
:thinking:  it might be worth seeing if the ray project would take a PR to make their builds multiarch too... :thinking:

Reply:
I'll check the rayproject CI.

Reply:
It is using `buidkite` for its CI.  It may not be simple as github action.